src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -1023,7 +1023,12 @@ module T = struct
       Or_error.try_with (fun () ->
           List.map cs ~f:(fun cmd ->
               let open Ledger in
-              User_command.to_verifiable ~ledger ~get ~location_of_account cmd
+              (* TODO: what are we supposed to do here? *)
```

Comment:
I don't think we need to do anything here. `Verifiable.create` does what's required